ts/features/bonus/cdc/screens/CdcBonusRequestSelectResidence.tsx

```diff
@@ -76,7 +102,10 @@ const CdcBonusRequestSelectResidence = () => {
             I18n.t("global.buttons.continue"),
             undefined,
             undefined,
-            !isResidentInItaly ?? false
+            cdcSelectedBonus.length !==
+              cdcSelectedBonus.filter(
+                b => isResidentInItaly[b.year] === "residentInItaly"
+              ).length
```

Comment:
```suggestion
            cdcSelectedBonus.every(
                b => isResidentInItaly[b.year] === "residentInItaly"
              )
```
If I got the meaning, this could be more explicit

Reply:
You're right, updated in [2aa8e69](https://github.com/pagopa/io-app/pull/3949/commits/2aa8e69a2a93fd42588218a3d1d5fa3324902196)